PlaygroundExamples/Signals in general/Filter.playground/Pages/Basic intro.xcplaygroundpage/Contents.swift

```diff
@@ -1,39 +1,25 @@
-//: Playground - noun: a place where people can play
-
-import UIKit
-
+import Foundation
 import ReactiveSwift
 import ReactiveCocoa
 import Result
 
+// Filter removes entries based on a condition.
+let citySignal = Signal<String, NoError>.pipe()
 
-let originalSignal = Signal<String, NoError>.pipe()
-
-// Map converts from one return value to another
-
-let newSignal = originalSignal.output.map { string -> Bool in
-    
-    if string == "myString" {
-        return true
-    }
-    return false
+//In this case we remove empty strings.
+let nonEmptyCitySignal = citySignal.output.filter{ text in
+    text != ""  //Only let text that is not empty pass through.
 }
 
-// the signal now returns Bool
-
-newSignal.observeValues { (value) in
-    print(value)
+nonEmptyCitySignal.observeValues { (text) in
+    print(text)
 }
 
+//These are not printed because they are empty.
+citySignal.input.send(value: "")
+citySignal.input.send(value: "")
 
-originalSignal.input.send(value: "testValue")
-originalSignal.input.send(value: "testValue 2")
-originalSignal.input.send(value: "myString")
-
-
-
-
-
-
-
-
+//These are printed because they are not empty.
+citySignal.input.send(value: "Oslo")
+citySignal.input.send(value: "Kairo")
```

Comment:
Kunne man her operert med et annet eksempel på filtreringa, feks "er det en gyldig by?" som man filtrerer ut på? Tenker da bare om man sjekker om stringen er .contains i et array vi definerer med gyldige byer 
empty String er jo som oftest representert med en optional, så kan være mulig uklarhet for folk som leser dette.